src/Aspire.Hosting.Sdk/SDK/Sdk.targets

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <ProjectCapability Include="DynamicFileNesting" />
     <ProjectCapability Include="DynamicFileNestingEnabled" />
+    <ProjectCapability Include="AspireOrchestration" />
```

Comment:
Wouldn't it be better to call it "AspireAppHost" to match the template naming convention?

Reply:
I think the thought process here is that it's added by the workload SDK which itself is added to the project by the `IsAspireHost` property in the project, and the workload SDK only exists to facilitate the delivery of the orchestration pieces (DCP) so calling it `AsprieOrchestration` is inline with that and separate from the `IsAspireHost` property.

Reply:
That was my main thought in using `AspireOrchestration` vs `AspireHost`, `AspireAppHost`, etc.; it seems as though the existing `Aspire` capability declared in the `Aspire.Hosting` targets is basically the AppHost capability, so having a distinct enough capability for the workload SDK made sense.